src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -121,17 +121,17 @@
     <value>Select color scheme...</value>
   </data>
   <data name="NewTabParentCommandName" xml:space="preserve">
-    <value>New Tab...</value>
+    <value>New tab...</value>
   </data>
   <data name="SplitPaneParentCommandName" xml:space="preserve">
-    <value>Split Pane...</value>
+    <value>Split pane...</value>
   </data>
   <data name="ApplicationDisplayNamePortable" xml:space="preserve">
-    <value>Terminal (Portable)</value>
+    <value>Terminal (portable)</value>
     <comment>This display name is used when the Terminal application is running in a "portable" mode, where settings are not stored in a shared location.</comment>
   </data>
   <data name="ApplicationDisplayNameUnpackaged" xml:space="preserve">
-    <value>Terminal (Unpackaged)</value>
+    <value>Terminal (unpackaged)</value>

```

Comment:
I believe we should leave these unchanged since they aren't UI elements per-se.

Reply:
Okaay.. I though it was UI related (cause that display phrase) 😓

Reply:
Yup those two. They're the application names as shown in the start menu.

"Unpackaged" and "Portable" are subtexts with an importance equal to "Terminal" itself. In my experience such words are capitalized. [StackExchange seems to agree](https://english.stackexchange.com/a/164663) and has an excellent example for this:
> Captain America (Untold Stories)

Reply:
(If you tried to revert these lines with your "merge branch" commit, it was not successful.)

Reply:
Oh I see. Now it's fixed